<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Wrangler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browsefileButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#browsefileButton{
color: white;
background-color: #3d3d3d;
border: TRANSPARENT;
border-radius: 10px;
}
#browsefileButton:hover{
background-color: #4b4b4b;
}</string>
    </property>
    <property name="text">
     <string>Browse File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wrangleButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>432</y>
      <width>583</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#wrangleButton{
color: white;
background-color: #3d3d3d;
border: TRANSPARENT;
border-radius: 5px;
}
#wrangleButton:hover{
background-color: #4b4b4b;
}</string>
    </property>
    <property name="text">
     <string>Wrangle</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
background-color: white;
border: 1px solid #3d3d3d;
border-radius: 5px;
}
#widget:hover{
border: 2px solid #3d3d3d;
}</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>2</y>
       <width>461</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit{
border: TRANSPARENT;
}</string>
     </property>
     <property name="placeholderText">
      <string>File to path</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
